charts/supergraph/values.yaml

```diff
@@ -0,0 +1 @@
+# template values
```

Comment:
Do we need this comment for some reason?

Reply:
Just didn't want to have an empty file